sw/airborne/subsystems/gps/gps_ubx.c

```diff
@@ -90,7 +102,7 @@ void gps_ubx_read_message(void)
       gps_ubx_time_sync.t0_tow_frac   = UBX_NAV_SOL_Frac(gps_ubx.msg_buf);
       gps_ubx.state.tow        = UBX_NAV_SOL_ITOW(gps_ubx.msg_buf);
       gps_ubx.state.week       = UBX_NAV_SOL_week(gps_ubx.msg_buf);
-      gps_ubx.state.fix        = UBX_NAV_SOL_GPSfix(gps_ubx.msg_buf);
+      //gps_ubx.state.fix        = UBX_NAV_SOL_GPSfix(gps_ubx.msg_buf);
```

Comment:
Fix can now grow to values 4 and 5. But this line was obviously not good. Fixed in new version.